Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -154,10 +155,28 @@ void GameList::ShowContextMenu(const QPoint&)
   }
   if (platform == DiscIO::Platform::WII_WAD)
   {
-    menu->addAction(tr("Install to the NAND"), this, SLOT(InstallWAD()));
+    QAction* wad_install_action = new QAction(tr("Install to the NAND"), menu);
+    QAction* wad_uninstall_action = new QAction(tr("Uninstall from the NAND"), menu);
+
+    connect(wad_install_action, &QAction::triggered, this, &GameList::InstallWAD);
+    connect(wad_uninstall_action, &QAction::triggered, this, &GameList::UninstallWAD);
+
+    connect(this, &GameList::EmulationStarted,
```

Comment:
Maybe extract this to a separate function (or at least make this a ranged loop over `{wad_install_action, wad_uninstall_action}`) to avoid duplicating very similar code twice?